doc/langref.html.in

```diff
@@ -8718,11 +8765,12 @@ test "@wasmMemoryGrow" {
       function from {#syntax#}std/builtin.zig{#endsyntax#}.
       </p>
       <p>Generally it is better to use {#syntax#}@import("std").debug.panic{#endsyntax#}.
-          However, {#syntax#}@panic{#endsyntax#} can be useful for 2 scenarios:
+          However, {#syntax#}@panic{#endsyntax#} can be useful for 3 scenarios:
       </p>
       <ul>
         <li>From library code, calling the programmer's panic function if they exposed one in the root source file.</li>
         <li>When mixing C and Zig code, calling the canonical panic implementation across multiple .o files.</li>
+        <li>When working with embedded systems where the standard library is not being used.</li>
```

Comment:
This point makes no sense as `@panic` calls `std.builtin.panic` which is either `std.builtin.default_panic` or a user supplied `@import("root").os.panic` or `@import("root").panic`.

Reply:
This was meant to address #1854 c):

> C) For `@panic`, shouldn't you add a third use case for embedded systems
when there is no standard infrastructure, such as you used in your
bare-bones OS for your Raspberry Pi 3+?

Looking at `@panic` it seems to call `builtin.panic` as you suggest (which is no longer namespaced under `std`). If you provide your own ` @import("root").os.pani`c or` @import("root").panic` then it seems to avoid using the `std.debug.paniclmpl` which I *think* is what the person who wrote #1854 was going after (whereas if you use `std.debug.panic` directly you'd obviously be referencing `std` and all that entails).

This line can certainly be removed or reworded if its not longer desirable.

Thanks for giving this a look :)

Reply:
Embedded projects can, and do, use the standard library still.